ffi/diplomat/cpp/docs/source/datetime_ffi.rst

```diff
@@ -69,7 +69,7 @@
 
 .. cpp:class:: ICU4XGregorianTimeFormatter
 
-    An ICU4X TimeFormatter object capable of formatting a :cpp:class:`ICU4XGregorianDateTime` as a string, using the Gregorian Calendar.
+    An ICU4X TimeFormatter object capable of formatting a :cpp:class:`ICU4XGregorianDateTime` as a string
```

Comment:
Nit: Is the input type still `ICU4XGregorianDateTime`?

Reply:
Yes, but we can give it more input types in the future